src/user-management/user-management.ts

```diff
@@ -137,7 +145,16 @@ const toQueryString = (options: Record<string, string | undefined>): string => {
 };
 
 export class UserManagement {
-  constructor(private readonly workos: WorkOS) {}
+  private jwks: ReturnType<typeof createRemoteJWKSet> | undefined;
+
+  constructor(private readonly workos: WorkOS) {
+    const { clientId } = workos.options;
+
+    // Set the JWKS URL. This is used to verify if the JWT is still valid
+    this.jwks = clientId
+      ? createRemoteJWKSet(new URL(this.getJwksUrl(clientId)))
+      : undefined;
```

Comment:
I have this conditional for backwards compatibility. I think the larger `clientId` refactor should also be a separate PR.